tools/influx_si/influx_si.xml

```diff
@@ -0,0 +1,91 @@
+<tool id="influx_si" name="influx_si" version="0.8.0+galaxy0" python_template_version="3.5">
+    <description>metabolic flux analysis</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <!--
+#silent sys.stderr.write("!!!! Cheetah Template Variables !!!!\n")
+#for k,v in $searchList[2].items()
+#silent   sys.stderr.write(" %s = %s\n" % (str(k), str(v) ))
+#end for
+    -->
+    <version_command>influx_s --version</version_command>
+    <command detect_errors="exit_code"><![CDATA[
+       #set $inp_all = [$input_main, ($prlexp.input_aux if $prlexp.input_aux else [])]
+       #if $si.s_i == 'i':
+         #silent $inp_all.append($si.input_ti)
+       #end if
+
+       mkdir outdir && cd outdir;
+
+       #for $inp in $inp_all:
+         #for $i in $inp:
+           ln -s '$i' '$i.name';
+         #end for
+       #end for
+       
+       influx_$si.s_i
+         @COMMAND_OPT@
+         #if $si.s_i == 'i' and $si.time_order:
+           --time_order='$si.time_order'
+         #end if
+         '${"' '".join([str($in_m.name) for $in_m in $input_main])}'
+         > 'influx_${si.s_i}.stdout.txt' 2>'influx_${si.s_i}.stderr.txt' ;
+       
+       for f in *.ftbl *.log *.err *.R *.kvh; do mv -f "\$f" "\$f".txt || true; done;
```

Comment:
What are you trying to do here and the lines below?

Reply:
Here, I'm adding extra '.txt', '.pdf' and `.RData` extensions to make Galaxy recognise the file types and to avoid name collisions. If I leave files as they are then `__name_and__ext` pattern makes that in my collections all files in series `f.ftbl`, `f.log`, `f.err`, `f.R`, `f.pdf`... all have the same name `f` so only one of them is kept in the collection. To keep them all, I use this renaming. If you have a better suggestion, I am all ears.